### ISSUE TITLE
Add Talk: Ellen

### ISSUE THREAD

**Issue by IrinaMonica** [dmahlow/gophercon.es#28]
**Title**
Learn Neural Networks With Go - Not Math! / Ellen Körbes

**Text**
Studying neural networks is a surefire way to end up fighting more math than you can shake a stick at. Wish you could learn about the likes of gradient descent and backpropagation in a language you actually understand—like Go? Then this one is for you. Code, not math! Algorithms, not logarithms!

**Bio**
Ellen is a developer advocate at Garden, and also an avid gopher, actively involved with Women Who Go, and responsible for the most comprehensive Go course in Portuguese. They’ve spoken at world-famous events, and at countless local meet-ups. Ellen is a proud recipient of a ‘Best Hair’ award.

[More Info](http://ellenkorbes.com/)
[Twitter](https://twitter.com/ellenkorbes)

![Ellen](https://user-images.githubusercontent.com/47850422/54952239-cd775b80-4f45-11e9-91f1-3bdd374ba99b.jpg)
